core/app/modules/import-export/assets/js/pages/export/export-kit/components/kit-content/components/go-pro-button/go-pro-button.js

```diff
@@ -0,0 +1,30 @@
+import Button from 'elementor-app/ui/molecules/button';
+
+import useLink from '../../../../../../../hooks/use-link/use-link';
+
+export default function GoProButton( props ) {
```

Comment:
Isn't it supposed to be a general component?

Reply:
Yes, it can be - moving it.

Reply:
Done.